spec/ParseUser.spec.js

```diff
@@ -2300,6 +2300,66 @@ describe('Parse.User testing', () => {
     });
   });
 
+  it('case insensitive login with username', async () => {
+    let user = new Parse.User();
+    user.set('username', 'test1');
+    user.set('password', 'test');
+    await user.signUp();
```

Comment:
Should be wrapped in expectAsync so we know it was successful.

Reply:
It fails for me if I change the code to `user.set('password', 'test1');`, because it throws an error if the signup fails. Same for the other `async/await`s.

Reply:
Maybe you can use a wrapper and bind it to null?

Reply:
Changed to expectAsync.